packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -74,11 +78,21 @@ export class WaferMap extends FoundationElement {
     @attr({ attribute: 'color-scale-mode' })
     public colorScaleMode: WaferMapColorScaleMode = WaferMapColorScaleMode.linear;
 
+    /**
+     * @internal
+     */
+    public workerOne!: Remote<MatrixRenderer>;
```

Comment:
I would prefer a different suffix than `One`. Does this member even need a suffix?

Reply:
Once we had 2 workers, meanwhile we deleted the second. Indeed it doesn't make sense now that we have only one to name it workerOne, I will deleted the suffix